gcloud_utils/ml_engine.py

```diff
@@ -116,6 +118,38 @@ def create_new_model(self, name, description='Ml model'):
             parent=self.parent, body=request_dict)
         return request
 
+    def export_model(self, model_name):
+        try:
+            with open('model.pkl', 'wb') as model_file:
+                pickle.dump(model_name, model_file)
+        except Exception as error:
+            logging.error(error)
```

Comment:
add log message displaying the model_path and indicating that a error occurred.
raise the Exception